influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -171,6 +183,7 @@ pub struct WriteBufferImplArgs {
     pub wal_config: WalConfig,
     pub parquet_cache: Option<Arc<dyn ParquetCacheOracle>>,
     pub metric_registry: Arc<Registry>,
+    pub plugin_dir: Option<PathBuf>,
```

Comment:
What's the plan here? Prior to this change we were just storing the python file as a string. Do we want the plugin dir to be mirrored to S3?

Reply:
For now this is just for local development (i.e. I have InfluxDB running on my laptop and I'm working on `some_plugin.py` file that I want the server to just read in quickly to run the test). Users will deploy plugins by using the API or CLI. Although if we find that users would rather have a directory and deploy files to that dir themselves, we could have that as an option.